src/transformers/file_utils.py

```diff
@@ -2100,7 +2101,13 @@ def get_list_of_files(
         token = HfFolder.get_token()
     else:
         token = None
-    return list_repo_files(path_or_repo, revision=revision, token=token)
+
+    try:
+        return list_repo_files(path_or_repo, revision=revision, token=token)
+    except HTTPError as e:
+        raise ValueError(
+            f"{path_or_repo} is not a local path or a model identifier on the model Hub. Did you make a typo?"
+        ) from e
```

Comment:
(nit) - should we maybe also display the error message `e`?

Reply:
It will be displayed in the stack trace with the `from e`.

Reply:
ah I see yeah!

Reply:
it's more correct to just display this for 404 errors no?